app/models/import.rb

```diff
@@ -19,10 +20,41 @@ def load_data
     FileStorage.default.get_file(file)
   end
 
+  def load_logs
+    return if log_file.blank?
+
+    FileStorage.default.get_file(log_file)
```

Comment:
Would it be a good idea to cache this for the next call? Since we save many times in the middle of an import, this could slow things down a lot. (Sorry I must have missed this the last time around.)

Reply:
Alternatively, we could load + write the logs on a thread so they don’t block.